src/components/subscribe/subscribe.scss

```diff
@@ -43,7 +45,10 @@
   inline-size: 100%;
   inset-block-start: 0;
   inset-inline-start: 0;
-  position: absolute;
+
+  .c-subscribe:not(.is-destroyed) & {
```

Comment:
So I'm guessing `&:not(:only-child)` did not work?

Reply:
@gerardo-rodriguez I did not, thanks!

If the previous sibling is `hidden`, then you should be able to use some form of sibling selector?

```scss
.c-subscribe__controls-ui:not([hidden]) ~ & {
  position: absolute;
}
```

Reply:
Oh, good idea, thanks @tylersticka! Let me try it. 👍🏽

Reply:
@tylersticka Genius, this should work! 🎉

Reply:
Yay, thanks, @tylersticka! 

https://github.com/cloudfour/cloudfour.com-patterns/pull/1882/commits/864b7de01a8e5ced1d29158f4b08dca97e6ee027